curvefs/src/client/inode_wrapper.cpp

```diff
@@ -93,7 +148,28 @@ CURVEFS_ERROR InodeWrapper::Sync() {
     return CURVEFS_ERROR::OK;
 }
 
-CURVEFS_ERROR InodeWrapper::Refresh() {
+void InodeWrapper::FlushAttrAsync() {
+    if (dirty_) {
+        LockSyncingInode();
+        auto *done = new UpdateInodeAsyncDone(shared_from_this());
+        metaClient_->UpdateInodeAsync(inode_, done);
+        dirty_ = false;
```

Comment:
dirty_=false should update in done?

Reply:
no, should be update here, so when new write come, it can be set dirty again